File: FitoGrano/WinForm/frMain.vb
Public Class frMain 
    ' Локация герба {600; 12}
    Private Sub frMain_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load
        Me.ptbGerb.Parent = Me.ptbFone
        AddHandler DevExpress.LookAndFeel.UserLookAndFeel.Default.StyleChanged, AddressOf Default_StyleChanged
    End Sub

    Private Sub tmrDateTime_Tick(sender As System.Object, e As System.EventArgs) Handles tmrDateTime.Tick
        Me.lbDateTime.Caption = Now.ToLocalTime
    End Sub

    Private Sub btnSecurityUsers_ItemClick(sender As System.Object, e As DevExpress.XtraBars.ItemClickEventArgs) Handles btnSecurityUsers.ItemClick
        frSecurityUsers.ShowDialog()
    End Sub

    Private Sub btnOrganizationStructure_ItemClick(sender As System.Object, e As DevExpress.XtraBars.ItemClickEventArgs) Handles btnOrganizationStructure.ItemClick
        frBooks_OrganizationStructure.ShowDialog()
    End Sub

    Private Sub BarButtonItem2_ItemClick(sender As Object, e As DevExpress.XtraBars.ItemClickEventArgs) Handles BarButtonItem2.ItemClick
        frAddEditOrganizationsRoleJob.ShowDialog()
    End Sub

    ' Событие изменения темы оформления
    Private Sub Default_StyleChanged(ByVal sender As Object, ByVal e As EventArgs)
        RegistryWrite(pref_ComplexSettings, "SkinName", DevExpress.LookAndFeel.UserLookAndFeel.Default.SkinName)
    End Sub

    Private Sub btnClassificationProducts_ItemClick(sender As Object, e As DevExpress.XtraBars.ItemClickEventArgs) Handles btnClassificationProducts.ItemClick
        frBooks_ClassificationProducts.ShowDialog()
    End Sub

    Private Sub btnResearchDirection_ItemClick(sender As Object, e As DevExpress.XtraBars.ItemClickEventArgs) Handles btnResearchDirection.ItemClick
        frBooks_ResearchDirection.ShowDialog()
    End Sub

    Private Sub btnResearchObjects_ItemClick(sender As Object, e As DevExpress.XtraBars.ItemClickEventArgs) Handles btnResearchObjects.ItemClick
        frBooks_ResearchObjects.ShowDialog()
    End Sub

    Private Sub btnCountryAgreements_ItemClick(sender As Object, e As DevExpress.XtraBars.ItemClickEventArgs) Handles btnCountryAgreements.ItemClick
        frBooks_CountryAgreements.ShowDialog()
    End Sub
End Class

File: FitoGrano/CS/WinForm/frMain.cs
// VBConversions Note: VB project level imports
using System.Collections.Generic;
using System;
using System.Linq;
using System.Drawing;
using System.Diagnostics;
using Microsoft.VisualBasic;
using System.Xml.Linq;
using System.Windows.Forms;
// End of VB project level imports

using FitoGrano;

namespace FitoGrano
{
	public partial class frMain
	{
		public frMain()
		{
			InitializeComponent();
			
			//Added to support default instance behavour in C#
			if (defaultInstance == null)
				defaultInstance = this;
		}
		
#region Default Instance
		
		private static frMain defaultInstance;
		
		/// <summary>
		/// Added by the VB.Net to C# Converter to support default instance behavour in C#
		/// </summary>
		public static frMain Default
		{
			get
			{
				if (defaultInstance == null)
				{
					defaultInstance = new frMain();
					defaultInstance.FormClosed += new System.Windows.Forms.FormClosedEventHandler(defaultInstance_FormClosed);
				}
				
				return defaultInstance;
			}
			set
			{
				defaultInstance = value;
			}
		}
		
		static void defaultInstance_FormClosed(object sender, System.Windows.Forms.FormClosedEventArgs e)
		{
			defaultInstance = null;
		}
		
#endregion
		// Локация герба {600; 12}
		public void frMain_Load(System.Object sender, System.EventArgs e)
		{
			this.ptbGerb.Parent = this.ptbFone;
			DevExpress.LookAndFeel.UserLookAndFeel.Default.StyleChanged += Default_StyleChanged;
		}
		
		public void tmrDateTime_Tick(System.Object sender, System.EventArgs e)
		{
			this.lbDateTime.Caption = System.Convert.ToString(DateTime.Now.ToLocalTime());
		}
		
		public void btnSecurityUsers_ItemClick(System.Object sender, DevExpress.XtraBars.ItemClickEventArgs e)
		{
			frSecurityUsers.Default.ShowDialog();
		}
		
		public void btnOrganizationStructure_ItemClick(System.Object sender, DevExpress.XtraBars.ItemClickEventArgs e)
		{
			frBooks_OrganizationStructure.Default.ShowDialog();
		}
		
		public void BarButtonItem2_ItemClick(object sender, DevExpress.XtraBars.ItemClickEventArgs e)
		{
			frAddEditOrganizationsRoleJob.Default.ShowDialog();
		}
		
		// Событие изменения темы оформления
		private void Default_StyleChanged(object sender, EventArgs e)
		{
			mdRegistrySetting.RegistryWrite(mdVariables.pref_ComplexSettings, "SkinName", DevExpress.LookAndFeel.UserLookAndFeel.Default.SkinName);
		}
		
		public void btnClassificationProducts_ItemClick(object sender, DevExpress.XtraBars.ItemClickEventArgs e)
		{
			frBooks_ClassificationProducts.Default.ShowDialog();
		}
		
		public void btnResearchDirection_ItemClick(object sender, DevExpress.XtraBars.ItemClickEventArgs e)
		{
			frBooks_ResearchDirection.Default.ShowDialog();
		}
		
		public void btnResearchObjects_ItemClick(object sender, DevExpress.XtraBars.ItemClickEventArgs e)
		{
			frBooks_ResearchObjects.Default.ShowDialog();
		}
		
		public void btnCountryAgreements_ItemClick(object sender, DevExpress.XtraBars.ItemClickEventArgs e)
		{
			frBooks_CountryAgreements.Default.ShowDialog();
		}
	}
}
